addons/account/models/account_move.py

```diff
@@ -2549,7 +2549,12 @@ def _autopost_draft_entries(self):
             ('date', '<=', fields.Date.today()),
             ('auto_post', '=', True),
         ])
-        records.post()
+        if records:
+            cr = self.env.cr
```

Comment:
Why this intermediate variable?

Reply:
Use `self._cr` and remove this variable?

Reply:
Done